README.md

```diff
@@ -159,697 +159,6 @@ Currently, the bake command supports building images from compose files, similar
 
 There is also support for custom build rules from HCL/JSON files allowing better code reuse and different target groups. The design of bake is in very early stages and we are looking for feedback from users.
```

Comment:
Probably could move the "introduction" part to be included in the `buildx` top-level command, but can do that in a follow-up.